rest_api/actions/automatic_review.py

```diff
@@ -154,29 +154,9 @@ def __init__(self, request):
             abort(404, 'No data found for run id %s.' % self.run_id)
         self.lane_reviewers = [AutomaticLaneReviewer(lane) for lane in lanes]
 
-    @property
-    def run_elements(self):
-        return self.eve_get('run_elements', run_id=self.run_id)
-
-    @property
-    def cfg(self):
-        return review_thresholds['run']
-
-    @cached_property
-    def reviewable_data(self):
-        data = self.eve_get('runs', run_id=self.run_id)
-        if data:
-            return data[0]
-        else:
-            abort(404, 'No data found for run id %s.' % self.run_id)
```

Comment:
@tcezard Isn't this check still worth having? A check to confirm that there is indeed data for the run?

Reply:
The check on the `lanes` endpoint at line 154 already does this.